### PR TITLE
Home Work 11

### PR DESCRIPTION
**Pull request by TanyaB777**

### CODE REVIEW

File: src/main/java/hw_11/Task1.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        integerMap.put("Igor", 35);
+        integerMap.put("Nadya", 55);
+        integerMap.put("Sergey", 32);
+        integerMap.put("Andrey", 33);
+        integerMap.put("Alex", 36);
+        integerMap.put("Zoya", 21);
+        integerMap.put("Evgen", 37);
+        integerMap.put("Nikolay", 33);
+
+        System.out.println(integerMap);
```
Comment (AntBelyi) [inline src/main/java/hw_11/Task1.java:21]: missed
Screenshot_2

File: src/main/java/hw_11/Task3.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+        queue.offer("red");
+        queue.offer("black");
+        queue.offer("white");
+        queue.offer("orange");
+        queue.offer("yellow");
+
+        System.out.println(queue);
+
+        String str = "";
```
Comment (AntBelyi) [inline src/main/java/hw_11/Task3.java]: "" can be removed

File: src/main/java/hw_11/Task3.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+        if (!queue.isEmpty())
```
Comment (AntBelyi) [inline src/main/java/hw_11/Task3.java]: please use { and } symbols - it is more readable

File: src/main/java/hw_11/Task4.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
+package hw_11;
+
+import java.util.Deque;
+import java.util.LinkedList;
+import java.util.concurrent.DelayQueue;
```
Comment (AntBelyi) [inline src/main/java/hw_11/Task4.java]: Please remove unused imports

File: src/main/java/hw_11/Car.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public int getYear() {
+        return year;
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_11/Car.java]: Please remove unused methods

File: src/main/java/hw_11/Task2.java [hunk 6]
```diff
@@ -0,0 +1,100 @@
+package hw_11;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Task2 {
+
+    private static Scanner scanner = new Scanner(System.in);
```
Comment (AntBelyi) [inline src/main/java/hw_11/Task2.java]: can be final(don't forget about const naming)

File: src/main/java/hw_11/Task2.java [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+    private static Map<String, Person> people = new HashMap<>();
```
Comment (AntBelyi) [inline src/main/java/hw_11/Task2.java]: can be final as well

File: src/main/java/hw_11/Task2.java [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+            return;
+
+        Person person = new Person(surname, name, age);
+
+        people.put(surname.toUpperCase(), person);
+
+        System.out.println("Ви щойно додали " + person + " людину");
+
+        ;
+        ;
```
Comment (AntBelyi) [inline src/main/java/hw_11/Task2.java]: these ; ; symbols can be removed?

File: src/main/java/hw_11/Task2.java [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+        System.out.print("Введіть ім'я: ");
+        String name = scanner.nextLine().trim();
+
+        System.out.print("Введіть вік: ");
+
+        String inputAge;
+        int age = 0;
+
+        do {
+            System.out.print("Введіть вік: ");
```
Comment (AntBelyi) [inline src/main/java/hw_11/Task2.java:75]: two times ask my age
Screenshot_3

File: src/main/java/hw_11/Task1.java (hunk 1 above)
Comment (TanyaB777) [reply src/main/java/hw_11/Task1.java:21]: fixed

File: src/main/java/hw_11/Task3.java (hunk 2 above)
Comment (TanyaB777) [reply src/main/java/hw_11/Task3.java]: fixed

File: src/main/java/hw_11/Task3.java (hunk 3 above)
Comment (TanyaB777) [reply src/main/java/hw_11/Task3.java]: fixed

File: src/main/java/hw_11/Task4.java (hunk 4 above)
Comment (TanyaB777) [reply src/main/java/hw_11/Task4.java]: fixed

File: src/main/java/hw_11/Car.java (hunk 5 above)
Comment (TanyaB777) [reply src/main/java/hw_11/Car.java]: fixed

File: src/main/java/hw_11/Task2.java (hunk 9 above)
Comment (TanyaB777) [reply src/main/java/hw_11/Task2.java:75]: fixed

File: src/main/java/hw_11/Task2.java (hunk 7 above)
Comment (TanyaB777) [reply src/main/java/hw_11/Task2.java]: fixed

File: src/main/java/hw_11/Task2.java (hunk 6 above)
Comment (TanyaB777) [reply src/main/java/hw_11/Task2.java]: fixed

File: src/main/java/hw_11/Task2.java (hunk 8 above)
Comment (TanyaB777) [reply src/main/java/hw_11/Task2.java]: fixed

**Pull request merged**